zerver/lib/gitter_import.py

```diff
@@ -218,59 +219,71 @@ def build_subscription(recipient_id: int, user_id: int,
         id=subscription_id)
     return subscription
 
-def convert_gitter_workspace_messages(message_data: GitterDataT,
+def convert_gitter_workspace_messages(gitter_data: GitterDataT, output_dir: str,
                                       zerver_subscription: List[ZerverFieldsT],
-                                      user_map: Dict[str, int]) -> ZerverFieldsT:
+                                      user_map: Dict[str, int], chunk_size: int=800) -> None:
```

Comment:
The chunk size used in [export.py](https://github.com/zulip/zulip/blob/master/zerver/lib/export.py#L829) is 1000. But when I used the chunk size as `1000` with the dataset of 15493 messages and 273 users, I had a successful gitter to zulip data converision, but while importing, I received a Memory error. Working with 800 as the `chunk_size` didn't have any issue. I think it would be good if we decide upon a proper `chunk_size`.